db/models.py

```diff
@@ -45,3 +51,56 @@ class MovieSession(models.Model):
 
     def __str__(self):
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
There is better to use settings.AUTH_USER_MODEL

Reply:
I agree, you should do this